CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/NotesViewTopComponent.java

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright 2010-2020 Australian Signals Directorate
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.notes;
+
+import au.gov.asd.tac.constellation.graph.Graph;
+import au.gov.asd.tac.constellation.graph.manager.GraphManager;
+import au.gov.asd.tac.constellation.plugins.reporting.GraphReportListener;
+import au.gov.asd.tac.constellation.plugins.reporting.GraphReportManager;
+import au.gov.asd.tac.constellation.plugins.reporting.PluginReport;
+import au.gov.asd.tac.constellation.views.JavaFxTopComponent;
+import org.openide.awt.ActionID;
+import org.openide.awt.ActionReference;
+import org.openide.awt.ActionReferences;
+import org.openide.util.Exceptions;
+import org.openide.util.NbBundle.Messages;
+import org.openide.windows.TopComponent;
+
+/**
+ * UI component associated with the Notes View.
+ * 
+ * @author sol695510
+ */
+@TopComponent.Description(
+        preferredID = "NotesViewTopComponent",
+        iconBase = "au/gov/asd/tac/constellation/views/notes/resources/notes-view.png",
+        persistenceType = TopComponent.PERSISTENCE_ALWAYS)
+@TopComponent.Registration(
+        mode = "explorer",
+        openAtStartup = false)
+@ActionID(
+        category = "Window",
+        id = "au.gov.asd.tac.constellation.views.notes.NotesViewTopComponent")
+@ActionReferences({
+    @ActionReference(path = "Menu/Experimental/Views", position = 500),
+    @ActionReference(path = "Shortcuts", name = "CA-N")})
```

Comment:
Not sure I'm a fan of switching this to Ctrl-Alt given all other view shortcuts are Ctrl-Shift. I think it would be good to change this back to Ctrl-Shift and change the letter instead (maybe A for annotate?)